Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -138,10 +140,12 @@ private void OnAttemptEmag(EntityUid uid, DoorComponent door, ref OnAttemptEmagE
         }
     }
 
-    private void OnEmagged(EntityUid uid, DoorComponent door, ref GotEmaggedEvent args)
+    private void OnDoorEmagged(EntityUid uid, DoorComponent door, ref GotEmaggedEvent args)
     {
-        if (!SetState(uid, DoorState.Emagging, door))
+        if (!SetState(uid, DoorState.Opening, door))
```

Comment:
```suggestion
        if (!SetState(uid, DoorState.Emagging, door))
```
Emagging the door uses other timings for the state changes than opening it normally, so you should keep this.